pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -138,26 +138,34 @@ def _configure_swa_callbacks(self):
             self.trainer.callbacks = [StochasticWeightAveraging()] + self.trainer.callbacks
 
     def configure_progress_bar(self, refresh_rate=None, process_position=0):
-        if os.getenv("COLAB_GPU") and refresh_rate is None:
-            # smaller refresh rate on colab causes crashes, choose a higher value
-            refresh_rate = 20
-        refresh_rate = 1 if refresh_rate is None else refresh_rate
-
         progress_bars = [c for c in self.trainer.callbacks if isinstance(c, ProgressBarBase)]
         if len(progress_bars) > 1:
             raise MisconfigurationException(
                 "You added multiple progress bar callbacks to the Trainer, but currently only one"
                 " progress bar is supported."
             )
         if len(progress_bars) == 1:
-            progress_bar_callback = progress_bars[0]
-        elif refresh_rate > 0:
+            return progress_bars[0]
+        elif refresh_rate is None or refresh_rate > 0:
```

Comment:
```suggestion
        if refresh_rate != 0
```
For some reason I prefer `if` over `elif` here, not sure why

Reply:
Yep fully agree, nice cleanup :)